src/EditorFeatures/Test/SolutionCrawler/WorkCoordinatorTests.cs

```diff
@@ -690,7 +690,7 @@ End Property
 
             var root = Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.ParseCompilationUnit(code);
             var property = root.FindToken(position).Parent.FirstAncestorOrSelf<Microsoft.CodeAnalysis.VisualBasic.Syntax.PropertyBlockSyntax>();
-            var memberId = (new Microsoft.CodeAnalysis.VisualBasic.VisualBasicSyntaxFactsService()).GetMethodLevelMemberId(root, property);
+            var memberId = Microsoft.CodeAnalysis.VisualBasic.VisualBasicSyntaxFactsService.Default.GetMethodLevelMemberId(root, property);
```

Comment:
We generally call it ".Instance" not ".Default"  .Default is when you can have multiple instances with different functionality.  Because functionality can't be changed with the syntax facts service, we don't need that naming here.